### ISSUE TITLE
not working with admin bro version 2.2.4

### ISSUE THREAD

**Issue by MohamedMohy** [SoftwareBrothers/admin-bro-theme-dark#2]
i tried to require and use it as shown in the readme file but still no change in the colors.

**Comment by wojtek-krysiak** [comment]
it was prepared for admin-bro v1.6 and haven't been updated to v2. Waiting in the backlog.

**Comment by 1ay1** [comment]
Any progress on this?

**Comment by wojtek-krysiak** [comment]
nope. you will have to do this on your own (the dark theme) by changing https://adminbro.com/Theme.html and (option) passing styles via https://adminbro.com/AdminBroOptions.html#Assets

**Comment by Cvmcosta** [comment]
Hello, can someone provide a better clarification on how i can get this working?

**Comment by NeniEmSu** [comment]
> Hello, can someone provide a better clarification on how i can get this working?

Maybe this would be helpful to you: [My answer](https://github.com/SoftwareBrothers/admin-bro-theme-dark/issues/1#issuecomment-758712917) from issue #1

**Comment by Cvmcosta** [comment]
@NeniEmSu Thanks a lot! I will try your solution.
